server/src/controllers/grades.ts

```diff
@@ -162,6 +163,47 @@ async function getFinalGradesFor(
   return finalGrades;
 }
 
+interface InstanceWithUsers extends CourseInstance {
+  Users: Array<User>
+}
+
+async function filterByInstanceAndStudentNumber(
+  instanceId: number,
+  assessmentModelId: number,
+  studentNumbersFiltered: Array<string> | undefined
+): Promise<Array<string> | undefined> {
+  const studentsFromInstance: InstanceWithUsers | null = await CourseInstance.findOne({
+    attributes: ['id'],
+    where: {
+      id: instanceId,
+      assessmentModelId
```

Comment:
Course instances are not supposed to be strictly tied to assessment models. They only have a reference to a default assessment mode, which likely isn't that important at all from a teacher's point of view, but may be of interest to students later. So this shouldn't require an assessment model ID.

Reply:
Good point, removed the assessment model id requirement here.